src/patternfly/components/Progress/progress.scss

```diff
@@ -12,6 +12,7 @@
   --pf-c-progress__bar--before--Opacity: .2; // one-off change in opacity to allow progress bar to automatically coordinate with the indicator color
   --pf-c-progress__indicator--Height: var(--pf-c-progress__bar--Height);
   --pf-c-progress__indicator--BackgroundColor: var(--pf-c-progress__bar--before--BackgroundColor);
+  --pf-c-progress__helper-text--MarginTop: calc(var(--pf-global--spacer--xs) - var(--pf-global--spacer--md));
```

Comment:
We haven't done this sort of thing much and I think it's worth considering how this should scale with the overall progress grid gap. We'll have the same question for the form component and anywhere else we do this.

The grid gap for the progress layout is represented by `--pf-c-progress--GridGap: var(--pf-global--spacer--md);` (`16px`). The helper text margin-top is `-12px` (whatever the gap is minus `12px`), so the gap above the helper text is `4px`, which is the `--xs` spacer value, which matches the form helper text 👍

So that means that if a user updates `--pf-c-progress--GridGap` to `24px` (`--lg` spacer), the gap above the helper text will be `12px`. And if the progress gap is `32px`, the help text is `20px`, and so on.

However, if the user updates the progress gap to `8px`, the helper text is now `-4px`, which is likely not what we want. Exhibits A and B:

<img width="167" alt="Screen Shot 2022-10-11 at 1 05 56 PM" src="https://user-images.githubusercontent.com/35148959/195166704-a738bb6a-3918-401f-9401-aa1b733025a0.png">

<img width="875" alt="Screen Shot 2022-10-11 at 12 58 07 PM" src="https://user-images.githubusercontent.com/35148959/195166707-e35966f5-641c-4255-bb64-12f49aac4ef2.png">

I guess the question is, is there a proportion to what the helper text gap should be? For example, if the overall gap is `16px` now, and the helper text gap is `4px`, should the helper text be 25% of the overall gap? This may work, but it will give us spacers that don't exist in our spacing system. For example, a `24px` overall gap would create a `6px` gap for the helper text, and `6px` isn't part of our main spacer vars (we have a special spacer for form control elements we use there though)

Or should the helper text gap be separate from the overall gap - meaning it's `4px` regardless whatever the overall gap is - and it's implied that they should be updated separately? Something like `calc(var(--overall-gap) * -1 + var(--helper-text-gap))`?

curious your thoughts @mcarrano @mmenestr @mceledonia

Reply:
@mcoker Hmm I'd probably say they should be updated separately and the gap between the progress bar and the helper text should remain 4px no matter what.